File: fern/snippets/tool-calling/typescript/file-upload.ts
import { Composio } from '@composio/core';
import path from 'path';

const composio = new Composio({
  apiKey: 'your-composio-api-key'
});

// Upload a local file to Google Drive
const result = await composio.tools.execute('GOOGLEDRIVE_UPLOAD_FILE', {
  userId: 'user-4235***',
  arguments: {
    file_to_upload: path.join(__dirname, 'document.pdf')  // Local file path
  }
});

console.log(result.data);  // Contains Google Drive file details

File: fern/snippets/tool-calling/python/file-upload.py
import os

from composio import Composio

composio = Composio(api_key="your_composio_key")

# Upload a local file to Google Drive
result = composio.tools.execute(
    slug="GOOGLEDRIVE_UPLOAD_FILE",
    user_id="user-1235***",
    arguments={"file_to_upload": os.path.join(os.getcwd(), "document.pdf")},  # Local file path
)

print(result)  # Print Google Drive file details
